### PR TITLE
Create a desktop version of the project

### PR DESCRIPTION
**Pull request by Rustamxon7**
I created 👨‍💻 a  **desktop version**  of the project in this branch using a media query. My website is designed for 

- mobile 📱
- mac os (_retina display_) 💻
- desktop 🖥

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -304,25 +311,289 @@ body {
...
+  border-radius: 0.8rem;
   width: 100%;
 }
 
 .btn-contact {
   display: inline-block;
   background-color: #fff;
 }
+
+@media only screen and (min-width: 1440px) {
```
Comment (arslanbisharat) [inline styles.css:337]: - [ ] The value for `min-width` property should be in range of `992px` - `1100px`

File: styles.css [hunk 2]
```diff
@@ -304,25 +311,289 @@ body {
...
+  .contact-element {
+    border-radius: 0.1rem;
+  }
+
+  .btn-contact {
+    margin-left: 16.5rem;
+  }
+}
+
+@media screen and (min-device-width: 1200px) and (max-device-width: 1600px) and (-webkit-min-device-pixel-ratio: 2) and (min-resolution: 192dpi) {
```
Comment (arslanbisharat) [inline styles.css:469]: - [ ] The value for `min-width` property should be in range of `992px` - `1100px`

File: styles.css [hunk 3]
```diff
@@ -165,68 +171,69 @@ body {
...
   align-items: center;
   list-style: none;
-  font-size: 13px;
+  font-size: 1.3rem;
 }
 
 .card-info .card-info-dot {
-  height: 8px;
+  height: 0.8rem !important;
+  width: 0.8rem !important;
```
Comment (arslanbisharat) [inline styles.css:179]: - [ ] Please don't use !important in CSS. We should keep our code clean.

File: index.html [hunk 4]
```diff
@@ -130,47 +138,47 @@ <h2 class="heading-secondary">Multi-Post Stories</h2>
...
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#" class="connect-btn">LET'S CONNECT</a>
+                <ul class="social-icons">
+                    <li><img src="images/GitHub.svg" alt="GitHub"></li>
+                    <li><img src="images/Linkedin-icon.svg" alt="Linkedin-icon"></li>
+                    <li><img src="images/Vector.svg" alt="Vector"></li>
+                    <li><img src="images/Twitter.svg" alt="Twitter"></li>
+                    <li><img src="images/Medium.svg" alt="Medium"></li>
+                </ul>
```
Comment (arslanbisharat) [inline index.html:155]: - [ ]The images are not rendering properly becaause it need ./ to specify the path.

File: index.html [hunk 5]
```diff
@@ -105,19 +111,21 @@ <h2 class="heading-secondary">Tonic</h2>
...
-                    <li>CANOPY </li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li class="input-text">Back End Dev</li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li class="input-text">2015</li>
+                    <li>Uber</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">Lead Developer</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">2018</li>
```
Comment (arslanbisharat) [inline index.html:123]: - [ ]The images are not rendering properly becaause it need ./ to specify the path.

File: index.html [hunk 6]
```diff
@@ -84,19 +88,21 @@ <h2 class="heading-secondary">Multi-Post Stories</h2>
...
-                    <li>CANOPY </li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li>Back End Dev</li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li>2015</li>
+                    <li>FACEBOOK</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">Back End Dev</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">2015</li>
```
Comment (arslanbisharat) [inline index.html:100]: - [ ]The images are not rendering properly becaause it need ./ to specify the path.

File: index.html [hunk 7]
```diff
@@ -61,19 +65,19 @@ <h2 class="heading-secondary">Tonic</h2>
...
-                    <li>CANOPY </li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li>Back End Dev</li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li>2015</li>
+                    <li>FACEBOOK</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">Full Stack Dev</li>
+                    <li> <img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">2015</li>
```
Comment (arslanbisharat) [inline index.html:78]: - [ ]The images are not rendering properly becaause it need ./ to specify the path.

File: index.html [hunk 8]
```diff
@@ -36,20 +41,19 @@ <h1 class="heading-primary">
...
                 <ul class="card-info">
                     <li>CANOPY </li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li>Back End Dev</li>
-                    <li><img src="/images/section2/Counter.svg" alt="dot" class="card-info-dot"></li>
-                    <li>2015</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">Back End Dev</li>
+                    <li><img src="/images/section-work/Counter.svg" alt="dot" class="card-info-dot"></li>
+                    <li class="input-text">2015</li>
```
Comment (arslanbisharat) [inline index.html:56]: - [ ]The images are not rendering properly becaause it need ./ to specify the path.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested,

Hi @Rustamxon7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [x] Please add .vscode to` .gitgnore`.
- [x] Please add .DS_STORE to` .gitignore`
- [ ] The background images are not as present in the design. Please stick to the design. 
            

- The hamburger should be gone on desktop version as present in the design. 
- The Item Navigation should be aligned to the right.
-  Your whole [page ](https://raw.githack.com/Rustamxon7/Microverse-portfolio-project/microverse-template-1-project-desktop-version/index.html)looks like mobile version not desktop. 

- [ ] Some icons/images are broken. Please have  a look on them. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: index.html (hunk 4 above)
Comment (Rustamxon7) [reply index.html:155]: I guess it's working fine!

Comment (Rustamxon7) [reply index.html:155]: Are you sure about that?

**Comment by Rustamxon7** [comment]
> # Changes Requested,
> Hi @Rustamxon7,
> 
> Good job so far!
> There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> You can also consider:
> 
> * [x]  Please add .vscode to` .gitgnore`.
> * [x]  Please add .DS_STORE to` .gitignore`
> * [ ]  The background images are not as present in the design. Please stick to the design.
> * The hamburger should be gone on desktop version as present in the design.
> * The Item Navigation should be aligned to the right.
> * Your whole [page ](https://raw.githack.com/Rustamxon7/Microverse-portfolio-project/microverse-template-1-project-desktop-version/index.html)looks like mobile version not desktop.
> * [x]  Some icons/images are broken. Please have  a look on them.
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

I used display: none to disappear hamburger from the desktop version. Isn't that good enough?

File: styles.css (hunk 1 above)
Comment (rloterh) [reply styles.css]: This issue is well resolved ✔️

File: styles.css (hunk 2 above)
Comment (rloterh) [reply styles.css]: This issue is well resolved ✔️

File: styles.css (hunk 3 above)
Comment (rloterh) [reply styles.css]: This issue is well resolved ✔️

File: index.html (hunk 4 above)
Comment (rloterh) [reply index.html:155]: - [ ] OPTIONAL
Hello team, kindly note that the image may render differently based on a particular platform or in different browers. Kindly ensure to resolve this in a more standardized way which should work on all platforms or browsers. Kindly consider this suggestion by the previous reviewer.

Comment (Rustamxon7) [reply index.html:155]: I didn't understand how to solve the problem. Can you help me with this?

Comment (Rustamxon7) [reply index.html:155]: I put some changes can you check it, please!

File: styles.css [hunk 9]
```diff
@@ -304,25 +311,158 @@ body {
...
+  border-radius: 0.8rem;
   width: 100%;
 }
 
 .btn-contact {
   display: inline-block;
   background-color: #fff;
 }
+
+@media only screen and (min-width: 992px) and (min-width: 1100px) {
```
Comment (rloterh) [inline styles.css:342]: - [ ] OPTIONAL
Various contents are not properly aligned on your desktop version, and thus your page is overlapping on this version. We actually want our design to closely follow the provided [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122)

**Your design:**
image

image

-Kindly endeavor to improve on those overlapping and inconsistent alignment aspects.

**Review by rloterh — APPROVED** [review]
Hi Rostam & Daniel, 

Great job for making the necessary changes recommended by the previous reviewer. However, there's always room for improvement.
Kindly consider my OPTIONAL comments

## Optional suggestions
_Please keep in mind that you, for best practices and following the project requirements and the design templates closely kindly consider these aspects as outlined in the below comments._

**_Kindly notice that these are required changes that I'm sorry they didn't show up in the earlier review._**

<br>
Your project is complete! Congratulations! 🥂There is nothing else to say other than... it's time to merge it 👍.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 9 above)
Comment (Rustamxon7) [reply styles.css:342]: Thank you for reviewing my project! I have a question for you. Can I add more media queries? For example on a 1080p monitor and a Mac monitor. The reason is that in the last review I added a media query for mac and told me to delete it, and put another one instead.

**Pull request merged**